src/styles/blocks/button.scss

```diff
@@ -0,0 +1,45 @@
+.button {
+  &__link {
+    box-sizing: border-box;
+    display: block;
+    font-size: 16px;
+    line-height: 27px;
+    text-align: center;
+    text-decoration: none;
+    color: #FFF;
+    padding: 15px 48px;
+    background-color: $main-text-color;
+    border: 2px solid $main-text-color;
+    border-radius: 8px;
+    transition: all 0.3s;
```

Comment:
You can move this to a variable, to minimize change to have a different transition time 

Because a good practice to have the same transition duration in a whole site